lib/src/constants.dart

```diff
@@ -5,6 +5,11 @@ final RegExp importExportDartPackageRegex =
 /// Regex used to detect all Sass import directives.
 final RegExp importScssPackageRegex = RegExp(r'''\@import\s+['"]{1,3}package:\s*([a-zA-Z0-9_]+)\/[^;]+''');
 
+/// Regex used to detect all Less import directives.
+final RegExp importLessPackageRegex = RegExp(r'[\t ]*@import[\t ]+(?:\(.*\)\s+)?"packages\/([a-zA-Z1-9_-]+)\/');
```

Comment:
```suggestion
final RegExp importLessPackageRegex = RegExp(r'@import[\t ]+(?:\(.*\)\s+)?"(?:packages\/|package:\/\/)([a-zA-Z1-9_-]+)\/');
```
- I think you can get rid of the first `[\t ]*` since this is a full-file search anyway (there likely is only ever going to be a newline before the import)
- Adjusted the package part so that it works with `package://` URIs, as well – I'm never used Less, but I was going off of https://pub.dev/packages/less_dart. If this is accurate, then we'd just need to add a test for this